src/components/Feeds/BusinessFeed.js

```diff
@@ -48,10 +120,14 @@ function BusinessFeed({ businesses, onSearch, selectedFilters, loadingState }) {
       {!initialLoad && hasResults && !searching && (
         <>
           <SimpleGrid columns={[null, 1, 2]} spacing={10}>
-            {businesses.map(business => {
+            {businesses.map((business, index) => {
               const formattedLocation = `${business.city ? business.city : ''}${
                 business.city && business.state ? ', ' : ''
               }${business.state ? business.state : ''}`;
+
+              if (index === 3) {
```

Comment:
As it stands currently, this logic will overwrite the 4th business in the list.

Looking at the production site, the fourth business in the list is "Cloud 9 Bakers", this business (on the staging site here https://5efcde45a23084000711282c--rebuild-black-business.netlify.app/businesses) is overwritten by the `<RegistrationPromo>` component and does not show in the feed.

We need to make sure that the injection of this promo does not affect the order that businesses show up in the feed. I think that this may be something that the Backend team will need to help us with. So hold tight here for a bit @Mhmdabed11 while we figure that out.

@elchris @llexical - Is the API the thing that is determining the number of results that are returned to the frontend? If so, can we, with this PR, reduce that number to 19, to allow us to properly inject this promo component in such a way that the 4th business is also rendered and pagination is not negatively affected?

Reply:
Oops, my bad. You are actually right, this will override the 4th result. The question is do we need to show the promo card on every page, and do we need to have an even number of cards? In both cases, if I return, in the conditional statement ‘if (index===3)’, both the promo code component and the ‘ResultCard’ component wrapped with a  react fragment, it will solve the problem of the overridden 4th result.

Reply:
@Mhmdabed11 no worries at all! Your line of thinking here is correct for sure. If we adjust this to output both the promo card, and the ResultCard we should be in good shape.

Additionally, we should reduce the `pageSize` parameter here https://github.com/Rebuild-Black-Business/RBB-Website/blob/ed5fb33ac7980db8afbc75cd6625153054f903e9/src/hooks/useSearch.js#L10 from `20` to `19` to make sure that the number of cards shown is always even.